app/assets/javascripts/vendor_invocations.js.coffee

```diff
@@ -1,6 +1,7 @@
 $(document).ready ->
   $('.datepicker').datetimepicker
-    altFormat: 'DD, MM dd, yy' 
+    format: 'dddd, MMMM D, YYYY'
+    altFormat: 'DD, MM dd, yy'
```

Comment:
So `altFormat` wasn't being honored because it came before `format`?

Reply:
You know, now that you mention it, altFormat is not a valid option. will remove.

Reply:
also, FYI even in cases where it is a valid option, it does not control the format of the field. So even if it was being "honored" it wouldn't have made the format appear differently.